src/calendar-heatmap.component.js

```diff
@@ -158,34 +158,33 @@ export class CalendarHeatmap extends Component {
       'year'
     );
 
+    const getSummary = (date) => {
```

Comment:
See if you can declare this function in utils

Reply:
@maneetgoyal sir,
1. This function is only meant to be used for `globalOverview` which filters data according to years and output an integrated summary, other that it don't have any uses. We are not using it in any other places. we shouldn't move it to any other places.
2. We are using class specific variables `this.props.data` which completely makes it class dependent. 
So in my conclusion we shouldn't do it.

Reply:
Whta is preventing you from passing this.props.data as input argument

Reply:
What about the usability sir , it is not being used anywhere.

Reply:
prefer maintainability more, move it